src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/OrganisationCreationRequestValidator.java

```diff
@@ -172,4 +177,14 @@ private void isDxAddressValid(DxAddressCreationRequest dxAddress) {
         }
     }
 
+    public static void isInputOrganisationStatusValid(String organisationStatus, String allowedStatus) {
+        List<String> validStatusList = asList(allowedStatus.split(","));
+        String orgStatus = removeAllSpaces(organisationStatus);
+        if (isBlank(orgStatus) || !validStatusList.contains(orgStatus.toUpperCase())) {
+            log.error(ERROR_MESSAGE_INVALID_STATUS_PASSED);
```

Comment:
component name is missing in the log

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/694/commits/81a62734a204788492e0d2d01ddca8354043025f